tests/foreman/cli/test_registration.py

```diff
@@ -224,3 +226,42 @@ def test_negative_global_registration_without_ak(module_target_sat):
         'Failed to generate registration command:\n  Missing activation key!'
         in context.value.message
     )
+
+
+def test_positive_custom_facts_for_host_registration(
+    module_sca_manifest_org,
+    module_location,
+    module_target_sat,
+    rhel8_contenthost,
```

Comment:
Use rhel_content, and parametrize on all host versions?
```suggestion
    rhel_contenthost,
```

Reply:
I don't thinks that's necessary, we are testing the functionality here which would be same for all host versions

Reply:
Even in that case, wouldn't it be preferable to use the general fixture and parametrize it with only one version? Time will come when RHEL8 will be obsolete and this fixture as well...